### PR TITLE
refactor: personal workspace refactor

### PR DESCRIPTION
**Pull request by laojun**
## What this PR does / why we need it:
refactor: personal workspace refactor

## I have checked the following points:
- [x] I18n is finished and updated by cli
- [x] Form fields validation is added and length is limited
- [x] Display normally on small screen
- [x] Display normally when some data is empty or null
- [x] Display normally in english mode

## Does this PR introduce a user interface change?

✅ Yes(screenshot is required)
image

## ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English |   refactor: personal workspace refactor           |
| 🇨🇳 中文    |   refactor: 个人工作台重构           |

## Does this PR need be patched to older version?
✅ Yes(version is required)

## Which issue(s) this PR fixes:

Fixes #

### CODE REVIEW

File: shell/app/org-home/pages/personal-home.tsx [hunk 1]
```diff
@@ -31,26 +50,263 @@ const PersonalHome = () => {
...
+          className: 'bg-white pb-0 px-4',
+        },
+      },
+      messageList: {
+        props: {
+          defaultLogo: <ErdaIcon type="tongzhi" disableCurrent size={28} />,
+        },
+      },
+    };
+  }, [listType, publicOrgs]);
```
Comment (McDaddy) [inline shell/app/org-home/pages/personal-home.tsx:281]: has dependency warning?

File: shell/app/config-page/components/scale-card/scale-card.tsx [hunk 2]
```diff
@@ -17,18 +17,22 @@ import React from 'react';
 import './scale-card.scss';
 
 const ScaleCard = (props: CP_SCALE_CARD.Props) => {
-  const { execOperation, data } = props;
+  const { execOperation, data, onClick, props: configProps } = props;
   const { list } = data || {};
```
Comment (McDaddy) [inline shell/app/config-page/components/scale-card/scale-card.tsx:21]: I think there no need to `|| {}`

File: shell/app/config-page/components/scale-card/scale-card.tsx [hunk 3]
```diff
@@ -17,18 +17,22 @@ import React from 'react';
...
+  const { align } = configProps || {};
```
Comment (McDaddy) [inline shell/app/config-page/components/scale-card/scale-card.tsx:22]: if require a default value, I think should be `left/right` not `{}`

Comment (McDaddy) [inline shell/app/config-page/components/scale-card/scale-card.tsx:22]: I think its a anti ts pattern behavior, type of `configProps` is not subset of `undefined`, then basically, no need add `|| {}`

File: shell/app/config-page/components/scale-card/scale-card.tsx [hunk 4]
```diff
@@ -17,18 +17,22 @@ import React from 'react';
...
       {map(list, (item, i) => {
         return (
           <div
             key={item.label}
             className="item text-normal shadow-card"
-            style={{ left: -8 * i }}
-            onClick={() => execOperation(item.operations?.click, item)}
+            style={{ left: align === 'right' ? 8 * (list.length - i - 1) : -8 * i }}
+            onClick={() => {
+              execOperation?.(item.operations?.click, item);
```
Comment (McDaddy) [inline shell/app/config-page/components/scale-card/scale-card.tsx:33]: seems type error exist

File: shell/app/config-page/components/base-list/base-list.tsx [hunk 5]
```diff
@@ -30,69 +29,99 @@ const List = (props: CP_BASE_LIST.Props) => {
...
             },
           };
         });
 
-        const { click, star } = item.operations || {};
+        const { star, ...restOp } = item.operations || {};
 
-        const { operationsOrder } = item.moreOperations || {};
-        const moreOperations = map(item.moreOperations?.operations, (action, key) => {
+        const moreOperations = map(item.moreOperations || [], (opItem, idx) => {
```
Comment (McDaddy) [inline shell/app/config-page/components/base-list/base-list.tsx:76]: no need ` || []` when use lodash map

File: shell/app/common/components/dropdown-select-new/index.tsx [hunk 6]
```diff
@@ -0,0 +1,244 @@
...
+            option={{ ...chosenItem }}
+            size={size}
+            onlyIcon={mode === 'simple'}
+            className={`p-0 seleted-item ${className}`}
+            switcher={
+              <span className="rounded-sm bg-default-06 text-default-8 px-2 py-0.5 ml-1">{i18n.t('dop:switch')}</span>
+            }
+          />
+        ) : (
+          <div>{i18n.t('dop:please choose')}</div>
```
Comment (daskyrk) [inline shell/app/common/components/dropdown-select-new/index.tsx:162]: Should not use dop namespace

File: shell/app/common/components/dropdown-select-new/index.tsx [hunk 7]
```diff
@@ -0,0 +1,244 @@
...
+  const useOption = option.children;
+
+  return (
+    <div className={'erda-dropdown-select-option-group  w-full'}>
+      <div
+        className="erda-dropdown-select-option-group-label flex items-center justify-between"
+        // onClick={() => setExpand(!expand)}
+      >
+        <div className="flex items-center text-xs p-1">{option.label}</div>
+        {/* <ErdaIcon type="down" className={`expand-icon flex items-center ${expand ? 'expand' : ''}`} size="16" /> */}
```
Comment (daskyrk) [inline shell/app/common/components/dropdown-select-new/index.tsx:233]: This icon not used?

File: shell/app/common/components/empty-holder/index.tsx [hunk 8]
```diff
@@ -43,10 +56,10 @@ export const EmptyHolder = ({
     'multi-line': true,
     relative,
   });
-  if (scene) {
+  if (scene && EmptyImgMap[scene]) {
     return (
       <div className={`scene-empty-holder inline-flex items-center justify-center bg-default-02 ${className || ''}`}>
-        <img src={`/empty-holder/${scene}.svg`} alt={`${scene}-empty-image`} />
+        <img src={EmptyImgMap[scene]} alt={`${scene}-empty-image`} />
```
Comment (daskyrk) [inline shell/app/common/components/empty-holder/index.tsx:62]: add `static` prefix in prod mode.

File: shell/app/config-page/components/scale-card/scale-card.tsx (hunk 3 above)
Comment (laojun) [reply shell/app/config-page/components/scale-card/scale-card.tsx:22]: configProps is a subset of undefined

**Pull request merged**